src/Frontend/Driver.hs

```diff
@@ -390,6 +413,7 @@ getTypeEnv path = do
                     & (names %~ (<> (modEnv ^. names)))
                     . (types %~ (<> (modEnv ^. types)))
                     . (classDecs %~ (<> (modEnv ^. classDecs)))
+                    . (classes %~ (<> (modEnv ^. classes)))
```

Comment:
Hrmr, does this mean class instances are exposed too? This code was originally copied from `Types.Infer`.

Reply:
Yes

Reply:
OK, this feels a little problematic, but I can live - I guess we can fix it when I sort out prelude loading for `compile`.